### ISSUE TITLE
About the PH2 dataset

### ISSUE THREAD

**Issue by lijincan11** [wurenkai/UltraLight-VM-UNet#27]
![QQ截图20240422214455](https://github.com/wurenkai/UltraLight-VM-UNet/assets/45137314/89e03076-8ca9-4f97-b3dd-15972a28836e)
Hello author, I downloaded the ph2 dataset from the connection you provided, and its structure is as follows. My question is, do I need to process it before running the Prepareuph2.py file? If I need to handle it, what are the training and testing sets, respectively?

**Comment by wurenkai** [comment]
Hi, since the PH2 dataset has a small number of cases, so in our work its mainly treated as an external validation.
The PH2 dataset (200 cases) is treated entirely as a test set. Hope my answer will help you.

**Comment by lijincan11** [comment]
Okay, thank you for your reply. If I want to reproduce, what steps should I take?


&nbsp;
***@***.***



&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "wurenkai/UltraLight-VM-UNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月22日(星期一) 晚上11:32
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [wurenkai/UltraLight-VM-UNet] About the PH2 dataset (Issue #27)





 
Hi, since the PH2 dataset has a small number of cases, so in our work its mainly treated as an external validation.
 The PH2 dataset (200 cases) is treated entirely as a test set. Hope my answer will help you.

**Comment by wurenkai** [comment]
Hi, you need to adjust the data format and use 'Prepare_PH2.py '. You can refer to [issue 8](https://github.com/wurenkai/UltraLight-VM-UNet/issues/8).

**Comment by ytuv** [comment]
> Hi, since the PH2 dataset has a small number of cases, so in our work its mainly treated as an external validation. The PH2 dataset (200 cases) is treated entirely as a test set. Hope my answer will help you.

Hello author, if PH2 is used as external validation and all are used as test sets, are the training dataset and validation set based on isic2017 or isic2018?

**Comment by wurenkai** [comment]
@ytuv The test set in the PH2 dataset was used for external validation only, which was trained on ISIC2017, and the process did not have a validation set.
